app/views/placements/support/organisations/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl"><%= t("organisations").capitalize %></h1>
```

Comment:
Not a strong opinion, but would it make more sense for the locale file to define the capital "O" on this, rather than us post-processing the translation?

```suggestion
    <h1 class="govuk-heading-xl"><%= t("organisations") %></h1>
```

```yaml
en:
  organisations: Organisations
```

Or if the key seems too clashy, I guess it could be `organisations_heading`?

Reply:
(Feel free to disagree and mark this as 'resolved')

Reply:
Sure!